File: 2-improved.js
let first_num = 2;
let second_num = 3;
let sum = sum_of_args(first_num, second_num);

function sum_of_args() {
    let sum = 0;
    for (i of arguments) {
        sum += i;

    } return sum;
}

console.log(sum);

File: 2-improved.py
first_num = 2
second_num = 3

def sum_of_args(*args):  # (*) - дает переменное количество аргументов
    sum = 0

    for i in args:
        sum += i
    return sum

print(sum_of_args(first_num, second_num))
